src/content-renderer.js

```diff
@@ -0,0 +1,31 @@
+import React from 'react';
+
+export const contentRenderer = function({node, styles, path, treeIndex, buttons}) {
```

Comment:
As `contentRenderer` is quite close to `nodeContentRenderer`, I'd prefer you change the name to `innerContentRenderer`.

Also, do you mind making this `export default ...` just to be more consistent with other files?

One last naming thing - please change the file name to `default-inner-content-renderer.js` to better represent its significance.